controllers/provider-aws/charts/provider-aws/templates/daemonset-custom-mtu.yaml

```diff
@@ -0,0 +1,30 @@
+{{ if eq .Values.gardener.seed.provider "aws" }}
+kind: DaemonSet
```

Comment:
It needs [tolerations](https://github.com/gardener/garden/blob/d7258fb99fbbc8f5a674c7459e04240a8ff094fd/charts/shoot-core/charts/kube-proxy/templates/kube-proxy-daemonset.yaml#L50-L56) and custom [priority class](https://github.com/gardener/gardener/blob/d7258fb99fbbc8f5a674c7459e04240a8ff094fd/charts/shoot-core/charts/kube-proxy/templates/kube-proxy-daemonset.yaml#L49) as `system-cluster-critical` can only be used in `kube-system` NS.

Reply:
but I am not using `system-cluster-critical`, or do you suggest that it needs to be `system-cluster-critical` ?

Reply:
@mvladev btw, after 1.11 you can use `critical` for any namespace: https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#marking-pod-as-critical